test/clients/installation_finish_test.rb

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require "y2firewall/clients/installation_finish"
+
+Yast.import "Service"
+
+describe Y2Firewall::Clients::InstallationFinish do
+  describe "#title" do
+    it "returns translated string" do
+      expect(subject.title).to be_a(::String)
+    end
+  end
+
+  describe "#modes" do
+    it "runs in installation" do
```

Comment:
np: You could say "it runs in installation and autoinstallation" (or something similar) and write only one test:

```ruby
excpect(subject.modes).to eq([:installation, :autoinst])
```

We can leave it for later.